src/main/kotlin/graphql/kickstart/tools/SchemaParserOptions.kt

```diff
@@ -88,10 +92,15 @@ data class SchemaParserOptions internal constructor(
             this.allowUnimplementedResolvers = allowUnimplementedResolvers
         }
 
+        @Deprecated("Use missingResolverDataFetcherProvider instead.", level = DeprecationLevel.WARNING)
```

Comment:
We can remove the `level` param since it's warning by default.

Reply:
```suggestion
        @Deprecated("Use missingResolverDataFetcherProvider instead.")
```